# Curriculum Vitae

## Education			        		
**BS, Honors Biology | McGill University (2019-2023)**
  - Graduated with First Class Honors
  - Honors Thesis: *Exploring Trametinib Resistance in Lung Adenocarcinoma: What Ablated Kinases Could Function Synergistically with MEK1/2 Inhibition to Prevent Cancer Recurrence.*

### Research Experience
**National Institutes of Health:**
*Postbaccalaureate Research Fellow - Brownell Lab (May 2023 - Present)*
- As the group's mouse colony manager, I validate and conserve the genetic makeup and phenotypes of hundreds of mice spanning 50 genetic strains. I am also working to create a mouse model of Merkel cell carcinoma.
- Concurrently, I also work on other research projects that bridge the gap between the bench and the clinic. These include exploring how the immune system can precipitate spontaneous regression in Merkel cell carcinoma, investigating the role of Gfi1 in Merkel cell differentiation, and drafting dermatology case reports.

**McGill University:**
*Honors Undergraduate Researcher - Dankort Lab (September 2023 - April 202_)*
- My project aimed to uncover ablated kinases that resensitize drug-resistant lung adenocarcinoma (LUAC) cells to trametinib, a MEK inhibitor drug.
- This involved conducting a kinase-targeting CRISPR screen on mouse-derived, trametinib-resistant LUAC cells with p53 loss and oncogenic Kras or Braf mutations.

**Harvard School of Public Health:**
*Research Assistant - Lu Lab (July 2022)*
- Prepared ARRDC1-mediated microvesicles (ARMMs) by transfecting HEK293 cells with a plasmid containing a shortened ARRDC1-Cas9 fusion protein targeting the GFP gene.
- After collecting and quantifying the microvesicles, recipient U2OS-GFP cells were treated with the ARMMs.
- Genome editing efficacy was determined via flow cytometry and sequencing; the use of these ARMMs resulted in a decrease in GFP-positive cells, demonstrating that gene editing likely occurred.

**Harvard School of Public Health:**
*Research Assistant - Lin Lab (May - June 2022)*
- Assisted with the development of FAVOR (Functional Annotation of Variants Online Resource), which gathers, outlines, and visualizes functional annotation data.
- Employed Python, R, and the command line to create a back-end functional annotation database for the GRCh37 / hg19 reference genome.
- Optimized database code to prioritize efficient memory consumption and low computing time; this process included correcting memory leaks and implementing techniques such as lazy loading and sparse data structures.

